%%% start preambling . . .  %%%
\documentclass{article}

% required 
\usepackage[hyphens]{url} % this wraps my URL versus letting it spill across the page, a bad habit LaTeX has

\usepackage{Sweave}
\usepackage{graphicx}
\usepackage{natbib}
\usepackage{amsmath}
\usepackage{textcomp}%amoung other things, it allows degrees C to be added
\usepackage{float}
\usepackage[utf8]{inputenc} % allow funny letters in citaions 
\usepackage[nottoc]{tocbibind} %should add Refences to the table of contents?
\usepackage{amsmath} % making nice equations 
\usepackage{listings} % add in stan code
\usepackage{xcolor}
\usepackage{capt-of}%alows me to set a caption for code in appendix 
\usepackage[export]{adjustbox} % adding a box around a map
\usepackage{lineno}
\linenumbers
% recommended! Uncomment the below line and change the path for your computer!
% \SweaveOpts{prefix.string=/Users/Lizzie/Documents/git/teaching/demoSweave/Fig.s/demoFig, eps=FALSE} 
%put your Fig.s in one place! Also, note that here 'Fig.s' is the folder and 'demoFig' is what each 
% Fig. produced will be titled plus its number or label (e.g., demoFig-nqpbetter.pdf')
% make your captioning look better
%\SweaveOpts{prefix.string=/Users/Deirdre/Documents/github/ospree/analyses/traitors/Figures/demoFig, eps=FALSE} 
\usepackage[small]{caption}

\usepackage{xr-hyper} %refer to Fig.s in another document
\usepackage{hyperref}

\setlength{\captionmargin}{30pt}
\setlength{\abovecaptionskip}{0pt}
\setlength{\belowcaptionskip}{10pt}

% optional: muck with spacing
\topmargin -1.5cm        
\oddsidemargin 0.5cm   
\evensidemargin 0.5cm  % same as oddsidemargin but for left-hand pages
\textwidth 15.59cm
\textheight 21.94cm 
% \renewcommand{\baselinestretch}{1.5} % 1.5 lines between lines
\parindent 0pt		  % sets leading space for paragraphs
% optional: cute, fancy headers
\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhead[LO]{Draft early 2022}
\fancyhead[RO]{Temporal Ecology Lab}
% more optionals! %

%\graphicspath{ {./DoseResponse_WriteupImages/} }% tell latex where to find photos 
\externaldocument[supp-]{Traitors_Manuscript_supp}

%%% end preambling. %%%

\begin{document}

%\renewcommand{\refname}{\CHead{}}%not sure what this was supposed to do 
\renewcommand{\bibname}{References}%names reference list 

\SweaveOpts{concordance=TRUE} % For RStudio hiccups


\title{Woody plant phenological responses are strongly associated with key functional traits}
\date{\today}
\author{Deirdre Loughnan$^1$, Faith A M Jones$^1$, Geoffrey Legault$^1$, Mira Garner$^2$,\\
Darwin Sodhi$^3$, Daniel Buonaiuto$^4$, Catherine Chamberlain$^5$, Ignacio Morales Castilla $^6$,\\
Ailene Ettinger$^7$,  and E M Wolkovich$^1$}
\maketitle 

$^1$ Department of Forest and Conservation, Faculty of Forestry, University of British Columbia, 2424 Main Mall
Vancouver, BC Canada V6T 1Z4. \\

$^2$  Arnold Arboretum of Harvard University, 1300 Centre Street, Boston, Massachusetts, USA;\\ \\

$^3$  Organismic \& Evolutionary Biology, Harvard University, 26 Oxford Street, Cambridge, Massachusetts, USA;\\

$^4$  XXXX \\

$^5$  XXXX\\

$^6$ Edificio Ciencias, Campus Universitario 28805 Alcalá de Henares, Madrid, Spain\\

$^7$  XXXX \\
%Fix numbers once I have everyone's affiliations

Corresponding Author: Deirdre Loughnan deirdre.loughnan@ubc.ca \\

%Full word count: \\
%Summary word count: \\
%Introduction word count: \\
%Materials and Methods word count: \\
%Results and figure legends word count: \\
%Discussion word count: \\


\newpage

\section{Summary}

Key Words: Budburst phenology, functional traits, Trees, climate change

\section{Introduction} %EMW16Aug2022: After we square away results (soon I think) -- let's tackle the methods, then move to intro

Invasive plants are shaping plant communities worldwide, filling vacant niches and displacing native plants through the novel community landscapes and changes in resource availability.

\begin{enumerate}
\item many studies have tried to predict species invasive potential based on functional traits and the links between traits, resource use, and growth strategies
\item the same traits we associate with native plants tracking changes in climate may also allow invasive species greater fitness and adaptive advantage to invasive species
\item traits that promote earlier growth in the spring = priority effects and less competitive temporal niche
\item understanding the links between species traits’, climate, and temporal niche will provide greater insight into likely responses of species fitness and community dynamics in response to climate change
\end{enumerate}

\begin{enumerate}
\item Functional traits: decades of work associating easy to measure traits related to growth and competition
\item drawing bread generalizable trends independent of species 
\begin{enumerate}
\item Early vs late - expect species like many invasive, with fast growth strategies, all along fast axes = high sea, high photosynthesis (LNC), small seeds
\end{enumerate}

\item Phenology is similarity a well studied traits not included in broader framework
\item invasives fill empty niche early in the growing season
\item studies of woody plants - focus on how app are responding to changes in climate and the effect of key abiotic cues on growth and phenology

\item In woody plants there are 3 key cues - chilling, forcing, photoperiod
\item We assume these cues would relate to broader growth strategies 
\begin{enumerate}
\item Particularly tradeoff in resource investment and chilling
\item Forcing vs risk frost
\item Photoperiod and photosynthetic potential 
\end{enumerate}
\item Relate cues to traits
\item Predictions, limitations of previous work
\item Our aims

\end{enumerate}

{\bf Some text for intro and methods... } \\ % There's probably two goals of trait ecology -- improve mechanistic understanding and effectively 'getting rid of species' 
A major goal of trait ecology has been to predict species-level characteristics by their traits alone; to do this, we need to be able to replace species distinctions in models with species' traits. To that aim, our models are designed to attribute phenological variation (days to budburst) to species' trait values, while allowing for residual variation explained by species (partially-pooled). When traits explain a significant portion of variation, species will explain only a small amount, and suggest we may be able to predict species for which we only have trait values. 
\\ While phenology in observational data is often considered highly variable, it is actually highly predictable in experiments where the major cues (forcing, chilling, photoperiod) are known and thus we focus on cues here. 

\section{Predictions}
%EMW4Sep2022: Thank you! In the end, I found I just need info on SLA, so added those requests below. 
\begin{enumerate}
\item Chilling Species with high chilling requirements will have traits associated with greater protection from harsh conditions and higher competitive abilities, but requiring longer periods to ac- cumulate chilling and days to budburst. %EMW4Sep2022: Side note -- I know these are not how you would add them, but a reminder to keep the organization consistent -- we're  generally talking about how traits relate to cues, not how cues relate to traits ... 
\begin{enumerate}
\item SLA should positively correlate with increasing chill requirements since leaves with large SLA are less frost tolerant %EMW4Sep2022: So the hypothesis is not that early species have leaves that can withstand frost, but that they are more okay to lose their leaves? I would double check with Cat that this makes sense to her, or ask her to point you towards the lit on it. 
\item Height should positively correlate since there will be more competition for light
\item Seed mass may negatively correlate, as later species have less time for seed development
\end{enumerate}

\item Forcing Species with low forcing requirements will express traits related to frost tolerance and high productivity through faster resource acquisition.
\begin{enumerate}
\item Leaves should have smaller SLA because smaller, thicker leaves are more frost tolerant • \item Shorter in height, as there is less competition for light prior to canopy closure
\item Seed mass may be greater as there is more time for seed development
\item LNC will be greater and is a proxy for photosynthetic rates
\end{enumerate}

\item Photoperiod Species with high photoperiod requirements will budburst later and should have traits associated with greater competitive abilities.
\begin{enumerate}
\item SLA should positively correlate with increasing photoperiod requirements
\item Height should positively correlate since there will be more competition for light
\item LNC will be lower in later budbursting species
\end{enumerate}
\end{enumerate}

\section{Methods - pre-outline methods}

For our analysis, we combined phenological data from the OSPREE database (Ettinger2020) with functional trait data from the TRY(Kattge2020) and BIEN (Enquist2016) trait databases. \\

%describe OSPREE breifly
The OSPREE database contains phenological data for woody, deciduous species from experiments of phenological cues. First published in 2019, this database has since been updated, and now includes the review of an additional 623 and 270 new publications from each of the following search terms:
\begin{itemize}
\item (budburst OR leaf-out) AND (photoperiod OR daylength) AND temperature* 
\item (budburst OR leaf-out) AND dorman*.
\end{itemize}
 From this subsequent review, we an additional 12 papers met our selection criteria. For additional information on the construction of the OSPREE database and methods of cue estimates, see (OSPREE). Our analysis used all available budburst data for our 37 focal species, with the data originating from 28 unique studies. \\ % If other ms are published first, do we need to also include this?

%Describe TRY and BIEN breifly 
Both TRY and BIEN are large databases compiling plant trait data across many individuals, species, and studies. We initially collected all available trait data for the 234 species for which there is budburst data in the OSPREE database. 

Trait data for ten functional trait was recieved from the TRY databases for 96 of our species (Table S1 - table of requested traits for each database). Additional trait Data was also obtained from the BIEN database using the BIEN R package (Maitner2017). All trait data were requested or downloaded in December 2018. For our analysis, we only included trait data from adult individuals with a minimum height of 1.42 m and we removed all data from experiments or growing in non-natural habitats. Traits were also grouped where appropriate, for example, with separate entries for SLA values with petioles, without petioles, and for which no petiole presence was specified were all categorized as simply SLA in our analysis (see Table S1). Duplicated data across the datasets were also removed (n = 434905). Finally, we subsetted the data to include only species for which we had a complete dataset for each species and trait. This resulted in a dataset of only 26 species and six functional traits. After performing a PCA, we further refined our trait selection, and only included traits that did not show strong correlations. In this analysis, the principle component explained 32.2\% of variation while the second explained 23.4\% of the variation (Fig. S1). Due to strong association between the SLA and LDMC leaf traits, and similarly between stem specific density (SSD) and height, we further reduced the number of traits in our analysis to include only height, seed mass, LNC, and SLA. By including only these four traits, we were able to increase the number of species we could include in our analysis as we had had at least one trait measurement for 37 species (height n = 47781, seed mass n = 281, LNC n = 3853, SLA n = 7656). Given the abundance of height data and overrepresentation of height measurements for six of our focal species, we randomly sampled 3000 height measurements for each of these species to include in our analysis (n = 27318). This reduces the effect of trait values from these frequently measured species from overwhelming the partial pooling effect in our model. In addition we excluded seed mass data from the HE Marx dataset from BIEN, as it consisted of only one value, making it challenging to include the study level effect in our model.\\ 

\subsection*{Joint model of trait and phenology}

To understand the implications of linking traits directly to cue responses, we developed a joint hierarchical Bayesian model. Our model is composed of two sub-models, a trait model and a phenology model, that are co-estimated and linked by a shared parameter. Since each trait varied in the number of studies in which it is included as well as the number of individuals for which it is measured, we chose to model each trait separately. The first part of the joint model is a hierarchical intercept only model where the response variable $Y_{i,j}$ is the observed trait value of species $i$ from study $j$, and is assumed to be normally distributed. We further assume that the observed trait value is composed of a ``grand'' species trait value $\alpha_{\text{trait},i}$ that is shared across all individuals of a species and that is independent of environment, a hierarchical grouping term on the intercept for study, $\alpha_{\text{study},j}$, to account for study-level differences in environment or observation methods, and random error. This results in the following sub-model for each trait:
%should we define every term in our model so we can be specific when we discuss them? I (Faith) usually do this, but our model (models?) is so long!
%Our model uses species-level trait values in our first model to predict species sensitivities to forcing, chilling, and photoperiod experimental cues. In addition to including partial pooling across species, the trait portion of the model includes a study level effect, thereby accounting for not only differences across species, but also the effects of methodological differences, and differences across habitats. The first model in our analysis calculates the latent variable that is then incorporated into the second phenology model. Values close to zero reflect small relationships between traits and cues values, while greater values represent high correlations between traits and phenological cues. This model was developed and validated using test data.{}
 % How much detail is needed - justify our approach or just include the model?
 %Do we include code for combining the effect of the grand mean with the species level effect?
\begin{equation}
\label{TraitsLine_main}
Y_{i,j} \sim \mathcal{N}( \mu_{i,j}, \sigma_{\text{trait}})
\end{equation}
where $\sigma_{\text{trait}}$ represents random error in the trait value (i.e., independent of study or species) and:
\begin{equation}
\label{TraitsLine_main2}
\mu_{i,j} = \alpha_{\text{trait},i}+\alpha_{\text{study},j}
\end{equation}
with:
\begin{align}
\boldsymbol{\alpha_{\text{trait}}} = \{\alpha_{\text{trait},1}, \ldots, \alpha_{\text{trait},n}\}^T & \text{ such that }
  \boldsymbol{\alpha_{\text{trait}}} \sim \mathcal{N}(\mu_{\alpha_{\text{trait}}},\sigma_{\alpha_{\text{trait}}}) \\
\boldsymbol{\alpha_{\text{study}}} = \{\alpha_{\text{study},1}, \ldots, \alpha_{\text{study},n}\}^T & \text{ such that }
  \boldsymbol{\alpha_{\text{study}}} \sim \mathcal{N}(0,\sigma_{\alpha_{\text{study}}}) \nonumber
\end{align}

Parameters $\mu_{\alpha_{\text{trait}}}$ and $\sigma_{\alpha_{\text{trait}}}$ represent the mean trait value across all species and the standard deviation in trait values between species, respectively. The mean effect of study is assumed to be centered at $0$ with standard deviation $\sigma_{\alpha_{\text{study}}}$.\\

The second part of the joint model is a hierarchical linear model where the normally distributed response variable $Z_{i,k}$ is the day of budburst for species $i$ experiencing forcing ($F_k$), chilling ($C_k$), and photoperiod ($P_k$). This sub-model is linked to the trait sub-model via the shared parameters $\alpha_{\text{trait},i}$, representing the ``grand'' trait values of species that are independent of study. The overall structure of the phenology sub-model is similar to that of (Ettinger2020), except species' responses to forcing ($\beta_{\text{force},i}$), chilling ($\beta_{\text{chill},i}$), and photoperiod ($\beta_{\text{photo},i}$) are treated not as single parameters but as a combination of parameters, a species-specific response that is independent of its trait value (e.g., $\alpha_{\text{force},i}$) and an effect of its trait value (e.g., $\beta_{\text{trait}.\text{force}}$) that is multiplied by $\alpha_{\text{trait},i}$ and does not differ between species. In other words, species responses to cues interact with their ``grand'' trait values, and we assume this interaction is independent of species identity. The phenology sub-model can thus be written as:

\begin{equation}
\label{phen_main}
Z_{i,k}  \sim \mathcal{N}( \mu_{i,k}, \sigma_{\text{pheno}})
\end{equation}
where $\sigma_{\text{pheno}}$ represents random error in budburst day and:
\begin{equation}
\label{phen_main2}
\mu_{i,k} = \alpha_{\text{pheno},i}+\beta_{\text{force},i} \times F_k + \beta_{\text{chill},i} \times C_k + \beta_{\text{photo},i} \times P_k
\end{equation}
with:
\begin{align}
\label{betaEq}
\beta_{\text{force},i} = \alpha_{\text{force},i} + \beta_{\text{trait}.\text{force}} \times \alpha_{\text{trait},i} \\
\beta_{\text{chill},i} = \alpha_{\text{chill},i} + \beta_{\text{trait}.\text{chill}} \times \alpha_{\text{trait},i} \nonumber \\
\beta_{\text{photo},i} = \alpha_{\text{photo},i} + \beta_{\text{trait}.\text{photo}} \times \alpha_{\text{trait},i} \nonumber 
\end{align} 
and all species-specific parameters are, as in the trait sub-model, given hierarchical structure whereby:
\begin{align}
\boldsymbol{\alpha_{\text{pheno}}} = \{\alpha_{\text{pheno},1}, \ldots, \alpha_{\text{pheno},n}\}^T & \text{ such that }
  \boldsymbol{\alpha_{\text{pheno}}} \sim \mathcal{N}(\mu_{\alpha_{\text{pheno}}},\sigma_{\alpha_{\text{pheno}}}) \\
\boldsymbol{\alpha_{\text{force}}} = \{\alpha_{\text{force},1}, \ldots, \alpha_{\text{force},n}\}^T & \text{ such that }
  \boldsymbol{\alpha_{\text{force}}} \sim \mathcal{N}(\mu_{\alpha_{\text{force}}},\sigma_{\alpha_{\text{force}}}) \nonumber \\
\boldsymbol{\alpha_{\text{chill}}} = \{\alpha_{\text{chill},1}, \ldots, \alpha_{\text{chill},n}\}^T & \text{ such that }
  \boldsymbol{\alpha_{\text{chill}}} \sim \mathcal{N}(\mu_{\alpha_{\text{chill}}},\sigma_{\alpha_{\text{chill}}}) \nonumber \\
\boldsymbol{\alpha_{\text{photo}}} = \{\alpha_{\text{photo},1}, \ldots, \alpha_{\text{photo},n}\}^T & \text{ such that }
  \boldsymbol{\alpha_{\text{photo}}} \sim \mathcal{N}(\mu_{\alpha_{\text{photo}}},\sigma_{\alpha_{\text{photo}}}) \nonumber
\end{align}

Parameters $\mu_{\alpha_{\text{pheno}}}$, $\mu_{\alpha_{\text{force}}}$, $\mu_{\alpha_{\text{chill}}}$, $\mu_{\alpha_{\text{photo}}}$ represent the mean budburst day, response to forcing, response to chilling, and response to photo period across all species, respectively. Parameters $\sigma_{\alpha_{\text{pheno}}}$, $\sigma_{\alpha_{\text{force}}}$ , $\sigma_{\alpha_{\text{chill}}}$ , $\sigma_{\alpha_{\text{photo}}}$ are the standard deviations between species.

Forcing, chilling, and photoperiod ($F_k$, $C_k$, $P_k$) were z-scored to account for differences in the scale of predictors across studies (Gelman2006) %what is this citation? Gelman and Hill - 2007?
, as well as differences in the natural units for the cues. We assumed parameters had weakly informative prior distributions (generally normal or half-normal distributions) that we obtained from a series of prior predictive checks where the objective was to produce a wide but also plausible range of trait and phenology values (e.g., budburst dates between days $0-365$). The joint model was coded in the Stan programming language (Stan citation) and fit to the trait and phenology data (see above) using the rstan package (version, citation). For all traits, model fits were deemed valid based on \textit{Stan's} diagnostic metrics, including no divergences across $1000$ iterations, high effective sample size (\textit{n\_eff}), and scale reduction factor $\hat{R}$ close to 1 across $4$ chains. We quantify 90\% uncertainty interval interval of posterior distributions using the highest probability density index.   

% add paragraph about priors
We selected our priors using our prior knowledge of functional trait diversity in deciduous species and cue responses. In doing so, we assume that stronger environmental cues will result in earlier budburst dates, as reflected by negative cue values. Priors of the phenology portion of our model reflect this, and are centered on negative values. 

% \begin{align*}
%\hat{trait_i} &= \mu_{grand_{sp}} + \alpha study_{study_i} \\
%\mu_{grand_{sp}} &= \alpha_{grand} + \alpha sp_{sp_i} \\
%\alpha_{grand}  & \sim N(0, \sigma_{grand})\\
%\alpha sp & \sim N(0, \sigma_{sp}) \\
%\alpha study& \sim N(0, \sigma_{study}) \\
%trait_i & \sim N(\hat{trait}_i, \sigma_{trait}) \\
%\vspace{1ex}\\

%\hat{pheno}_i  &= \alpha pheno_{sp_i} + \beta force_{sp_i} * Forcing_{i} + \beta photo_{sp_i}  * Photo_{i} + \beta chill_{sp_i} * Chill_{i} \\
%\beta force_{sp} &= \alpha force_{sp} + \beta trait.force * \alpha sp_{sp}\\
%\beta chill_{sp} &= \alpha chill_{sp} + \beta trait.chill * \alpha sp_{sp}\\
%\beta photo_{sp} &= \alpha photo_{sp} + \beta trait.photo * \alpha sp_{sp}\\
%\alpha pheno & \sim N(\mu_{pheno}, \sigma_{pheno}) \\
%\alpha force& \sim N(\mu_{force}, \sigma_{force}) \\
%\alpha chill & \sim N(\mu_{chill}, \sigma_{chill}) \\
%\alpha photo & \sim N(\mu_{photo}, \sigma_{photo}) \\
%pheno_{i} & \sim N(\hat{pheno_i}, \sigma_{pheno}) \\
%\end{align*}

%As such, we model each trait individually using the same model specified above, but with the appropriate priors for each trait. Priors were tested using prior predictive checks. All analyses were done in Stan (version) using the rstan package (version) in R (version). 

% I wrote this up in case we want to include it
Finally, we used a phylogenetic generalized least-squares regression model (PGLS) to test the relationship between day of budburst and individual traits. This analysis allowed us to test for phylogenetic non-independence in the phenology-trait relationship (Freckleton2002). We obtained a rooted phylogenetic tree by pruning the tree developed by (Smith2018a) and performed the PGLS analysis using the mean trait values and mean posterior estimates of the cue responses from our joint model. The PGLS was run using the "Caper" package in R (Orme2013). \\


\section{Results}
<<label=runcode1,  echo=FALSE>>=
#setwd("../../analyses/traits")

source("../../analyses/traits/Rfiles/SM_modelOutputTable.R")

@

%EMW4Sep2022: Keeping track of figures ... and making some notes 
\iffalse
sigmaDensities (current Fig 1) 
cuetraits (current Fig 2) 
slopes (current Fig 3) -- can you label briefly what each of the three top panels shows (I actually recommend starting an issue on this so we can iterate on it quickly)? The middle is when the trait does not matter? But the other two show what -- are they same outcome with just aquisiitive vs. conservative flipped?
TraitDistributions (current Fig 4)
\fi

\begin{enumerate}
\item Quick overview of models again ... (but try not to repeat ourselves from methods)
\begin{enumerate}
\item Our models jointly estimated species traits and the phenological cues to understand connections between phenology (specifically days to budburst) and other plant traits.
\item Because we drew on large trait databases for our analyses we partitioned trait variation due to species versus study-level effects. We then used the species-level estimates of traits (ie., estimates of species with study removed) to examine how traits predicted phenological cues, which in turn predict budburst, while allowing for residual variation in cues to be explained by a species effect. %EMW4Sep2022: This is a good transition to next section
\item (Move to methods): This model is a departure from current efforts in several ways: first, while these steps would normally be done separately, our models allow them to be jointly estimated at once. This allows separating out the trait versus residual species effect; second, we focus on phenological cues as predictors for day of budburst and thus allow traits to influence each cue. 
\end{enumerate}


\item Review of trait model part ... comparing study variation to species level variation
\begin{enumerate}

\item Across our models, we found important study-level variation across traits. 
\item In our height model, study level variaition was greater (\Sexpr{htSigmaStudy}) than species level variation (\Sexpr{htSigmaSp}) (Fig. \ref{fig:sigmaDensities}). 
\item In contrast, study level variation in our SLA model was approximately half the value of the species level variations (\Sexpr{slaSigmaSp} and \Sexpr{slaSigmaStudy} respectivley).
\item In the seed mass and LNC models it was considerably less than that of the species level variation (seed mass species level variation: \Sexpr{seedSigmaSp} and study level variaion: \Sexpr{seedSigmaStudy}; LNC species level variation: \Sexpr{lncSigmaSp} and study level variation \Sexpr{lncSigmaStudy})  (Fig. \ref{fig:sigmaDensities}). 
% \item ...provided estimates of species-level traits with lower uncertainty than simply using the raw data (Fig 3).  DL: what do you mean by this? Report the muSp and CI for species that we have good estimates for and for species we have bad estimates for?
%\item For most (?) traits we estimated average trait values similar to averages from the raw data, but for some (examples? Maybe horse chestnut and height? Pick one with lots of data) these large study-level effects impacted species-level mean estimates. % not sure we need this...
\end{enumerate}

\item Very short paragraph reminding audience of the model and connecting figure on cue-trait relationships. 
%EMW4Sep2022: first fig ref should be TraitDistributions (if we keep in main text; did we decide? ... just now I like it)
%DL5Sep2022: We didn't decide, but I would be happy to keep it in the main text and move the model fit (fig 4) to the supplement.
\begin{enumerate}
\item We next combined these trait-level estimates (Fig. \ref{fig:cuetraits}) with data on forcing, chilling and photoperiod values from a large meta-analysis of experiments to test if traits (height, SLA, seed mass, LNC) related to phenological cues and species budburst dates (Fig. \ref{fig:slopes}). 
\item For some traits we found much of the variation in phenological cues across species could be explained by trait variation (see panel a-f and j-l in Fig \ref{fig:cuetraits}), while for others traits were weak predictors (see g-i in Fig \ref{fig:cuetraits}). 
\end{enumerate}

%Dl - keeping the below comment to keep in in my thoughts
% EMW2Jul  -- I wonder if 'responses to changes in cues level' or similar is the best way to explain all of this? We may sometimes want to say that traits were not predictive of responses (see above)? Or I think I like what you had but I cut: differed in whether they led to greater advances (earlier budburst) with increasing cues. I think things like 'taller species had larger responses to chilling' also work. 
\item Discuss how Height and SLA were partially in line with predictions %EMW16Aug2022: Can you jot down own predictions at end of intro so I can refer back?
\begin{enumerate}
\item Across traits, height, SLA, and LNC showed the greatest responses to changes in cues level, though the direction of responses were consistent across cues for only LNC and not height or SLA (Fig. \ref{fig:cuetraits} a-f and j-l). 
%DLSept6 I can change the order of the traits in the plot if this makes it into the final version so it will be a-i
\item Taller species had extremely weak responses to forcing (\Sexpr{htBFSpMean} m per standardized forcing; 90\% uncertainty intervel: \Sexpr{lower_htBFSpMean}, \Sexpr{upper_htBFSpMean}).
\item However, as we predicted, taller species had larger responses to chilling (\Sexpr{htBCSpMean} m per standardized chilling; 90\% uncertainty interval interval: \Sexpr{lower_htBCSpMean}, \Sexpr{upper_htBCSpMean})) and photoperiod (\Sexpr{htBPSpMean} m per standardized photoperiod; 90\% uncertainty intervel: \Sexpr{lower_htBPSpMean}, \Sexpr{upper_htBPSpMean}, both of which are cues that would generally lead to later budburst and fit within an overall conservative growth strategy. 
\item High SLA species, which produce thin leaves with a relatively lower investment in leaf mass, had weaker forcing cues, contrary to our predictions, resulting in delayed budburst as forcing cues increased ((\Sexpr{slaBFSpMean} mm\textsuperscript{2}/mg per standardized forcing; 90\% uncertainty intervel: \Sexpr{lower_slaBFSpMean}, \Sexpr{upper_slaBFSpMean}). However, as we predicted, chilling cues were also weak (\Sexpr{slaBCSpMean} mm\textsuperscript{2}/mg per standardized chilling; 90\% uncertainty intervel: \Sexpr{lower_slaBCSpMean}, \Sexpr{upper_slaBCSpMean}). Across the range of trait values, low SLA (conservative) species did generally have a greater advance in budburst in response to chilling than high SLA (acquistive speies) (\ref{fig:cuetraits} b), again following the predicted gradient in growth strategies.
\item While our model did estimate stronger photoperiod cues (\Sexpr{slaBPSpMean} mm\textsuperscript{2}/mg per standardized photoperiod; \Sexpr{lower_slaBPSpMean}, \Sexpr{upper_slaBPSpMean}) as photoperiod increased, the effect on budburst dates were negligible (\ref{fig:cuetraits}), suggesting no relationship between SLA and photoperiod. 
%DL6Sept: the use of the word stronger might be confusing given that the bands are pretty flat, would it be less confusing to say "resulting in delaying effects"?
\item We found that species that produce leaves with high nitrogen content, which relate to high photosynthetic rates, showed weaker trait responses to all three cues, resulting in delayed budburst when trait effects were accounted for  (\Sexpr{lncBFSpMean} mg/g per standardized forcing; 90\% uncertainty intervel: \Sexpr{lower_lncBFSpMean}, \Sexpr{upper_lncBFSpMean}; \Sexpr{lncBCSpMean} mg/g per standardized chilling; 90\% uncertainty intervel: \Sexpr{lower_lncBCSpMean}, \Sexpr{upper_lncBCSpMean}; and \Sexpr{lncBPSpMean} mg/g per standardized photoperiod; 90\% uncertainty intervel:\Sexpr{lower_lncBPSpMean}, \Sexpr{upper_lncBPSpMean}. These findings are partially in line with our predictions as we predicted that LNC would have a weak relationship with chilling, but contrary to our predictions, increases in forcing and photoperiod also produced weaker, delaying effects on budburst.  


\end{enumerate}
\item ...But Seed mass 
\begin{enumerate}

\item Of our four traits, seed mass had the weakest responses across the three cues (Fig. \ref{fig:cuetraits} g-i).
\item Contrary to our predictions, we did not find a relationship between seed mass and cue responses for forcing ((\Sexpr{smBFSpMean} mg per standardized forcing; \Sexpr{lower_smBFSpMean}, \Sexpr{upper_smBFSpMean})) or photoperiod and only a moderate responses to chilling (\Sexpr{smBCSpMean} mg per standardized chilling; \Sexpr{lower_smBCSpMean}, \Sexpr{upper_smBCSpMean}). \Sexpr{smBPSpMean} mg per standardized  photoperiod; \Sexpr{lower_smBPSpMean},  \Sexpr{upper_smBPSpMean}).
\end{enumerate}
\end{enumerate}

\begin{figure}[h!]
    \centering
 \includegraphics[width=\textwidth]{..//..//analyses/traits/figures/sigmaSpStudy.pdf} 
   \caption{Comparison of the posterior densities for the species level variation and study level variation of a. height, b. SLA, c. LNC, and d. seed mass.} 
    \label{fig:sigmaDensities}
\end{figure}

\begin{figure}[h!]
    \centering
 \includegraphics[width=\textwidth]{..//..//analyses/traits/figures/cuetrait_wtrend_maintext.pdf} 
    \caption{Estimated trait values for height (a-c), SLA (d-f), log10 Seed mass (g-i), and LNC (j-l), correlated against species level cue responses to forcing (a, d, g \& j), chilling (b, e, h \& k), and photoperiod cues (c, f, i \& l). Parameters were estimated using our joint trait-phenology model, with the grey line depicting the mean linear relationship between estimated trait effects and the slope of the cue response. The 50\% confidence interval is depicted by the pink band.  The species depicted in Fig \ref{fig:slopes} are highlighted in each panel, with the acquisitive species shown in green, and the conservative species, shown in purple.}
    \label{fig:cuetraits}
\end{figure}

\begin{figure}[h!]
    \centering
 \includegraphics[width=\textwidth]{..//..//analyses/traits/figures/slopesConsAcqu_maintext_samespp.pdf} 
    \caption{Comparisons of estimated cue responses of a representative species with a trait value associated with acquisitive growth strategies, shown in green, or conservative growth strategies, shown in purple. Panels a. to c. represent a conceptual framework for interpreting trati responses. a) If species with traits associated with conservative (shown in dark purple) and acquisitive growth (shown in darker green) have stronger cue responses and greater advances in budburst date with increasing environmental cues, this will produce a steeper response (or slope) of the full model in comparison to the cue only (trait effect = 0) model. Furthermore, species with acquisitive traits are predicted to budburst earlier than species with more conservative traits. b) Functional traits that have no relation to budburst phenology will result in a trait effect estimated at or close to zero and result in no difference in the slopes of the full model relative to the cue only model. c) Traits with a positive estimated effect will result in the full model having a weaker budburst response to increasing environmental cues, leading to later budburst dates than estimated using the cue only model in which trait effects are zero. Budburst dates of species with traits associated with acquisitive growth will also be later if our estimates do not support the predicted gradients in growth strategies. Some traits strongly predicted phenological responses to cues, as seen by comparing the model estimates including trait (darker bands epresent the 50\% uncertainty interval for the posterior cue estimates for the full model) versus without (lighter bands represent the 50\% uncertainty interval interval for the posterior cue estimates with a trait effect of zero), which is especially obvious in the species with the higher trait values. Points represent the raw budburst data for each respective species. We show the full range of all treatment values (x axis) though most species cover only a portion of this range. } 
        \label{fig:slopes}
\end{figure}
    
% \begin{figure}[h!]
%     \centering
%  \includegraphics[width=\textwidth]{..//..//analyses/traits/figures/slopesConsAcqu_supp.pdf} 
%     \caption{Comparisons of estimated cue responses of a species with an trait value associated with acquisitive growth strategies, shown in green, or conservative growth strategies, shown in purple. Associations between seed mass and forcing, chilling, and photoperiod are depicted on panels a to c and associations between LNC and each cue in panel d to f. The green points represent the budburst data for \textit{Populus tremula}, a relatively small seeded species, while the green points are budburst data of the large seeded species, \textit{Aesculus hippocastanum}. Dark bands represent the 50\% credible interval for the posterior cue estimates for the full model. Opaque bands represent the 50\% credible interval for the posterior cue estimates with a trait effect of zero. The negative value of the seed mass model's slope for each cue produces a more negative effect on the day of budburst when seed mass is included in the model. This suggests that trees that produce large seeds advance their budburst dates at a higher rate to increasing cues (a-c). The effect of seed mass however, is relatively small compared to that observed from other traits. Estimates of the cue responses in our LNC model were all positive and produced more positive slopes in the full model. This indicates that high SLA values are less responsive in their budburst to increasing forcing, chilling, and photoperiod values (d to f). The greater effect of slopes on taller trees and high SLA species is a artifact of the trait value itself being larger and not a reflection on the magnitidue of the response.}

    
    % a smaller advance in budburst with increasing forcing when height effects were included % I don't fully understand this previous or the next sentence -- it reads as though we're contrasting (do we have models without height we plan to report?) ... wait, wait, I get it! You're talking about the counterfactual shown in Fig 1. I think we just keep this in a caption?s
%     \label{fig:slopes}
% \end{figure}



\begin{figure}[h!]
    \centering
 \includegraphics[width=\textwidth]{..//..//analyses/traits/figures/FourTraitFit_37spp_wp.png} 
    \caption{Comparisons of estimated model fits and raw data from joint models of trait effects on budburst phenological cues for 37 species of woody deciduous plants. Four functional traits -- a. height, b. SLA, c. seed mass, and d. LNC -- were modelled individually, with the calculated trait value being used to jointly model species responses to standardized chilling, forcing, and photoperiod cues. Model posteriors are shown in black, with the thicker line depicting the 66\% interval and the thinner black line the 97\% interval. Overall species level model posterior distributions were well aligned with the raw data, shown in red, and the species level means from the raw data, denoted as a purple stars.}
    \label{figure:TraitDistributions}
\end{figure}


\section{Discussion}

% Scrape

%\item Functional traits did influence the variation in the timing of budburst date in response to forcing, chilling, and photoperiod cues across species
%\begin{enumerate}
%\item but varied by trait and species whether responses in cue slopes followed the predicted gradients in trait values
%\item strongest effects for chilling 
%\item weakest effects observed for photoperiod
%\end{enumerate}
%\item The strength of species cue responses were associated with species trait values: referencing Fig. \ref{fig:cuetraits}
%\begin{enumerate}


\end{document}